ami/trainers/image_vae_trainer.py

```diff
@@ -81,9 +84,11 @@ def train(self) -> None:
                     dist_batch, Normal(torch.zeros_like(dist_batch.mean), torch.ones_like(dist_batch.stddev))
                 )
                 loss = rec_loss + self.kl_coef * kl_loss.mean()
+                self.logger.log("image_vae_loss", loss)
```

Comment:
このようにすると、`image_vae`というタブの中に`loss`がまとめられて見やすくなると思います。
```suggestion
                self.logger.log("image_vae/loss", loss)
```

Reply:
https://github.com/MLShukai/ami/pull/158/commits/f8c22ce7210ac142cc6a715c08b8710e7e9f63c4
修正しました